src/pystatis/http_helper.py

```diff
@@ -18,8 +18,8 @@
 
 logger = logging.getLogger(__name__)
 
-JOB_ID_PATTERN = re.compile(r"\d+-\d+_\d+")
-JOB_TIMEOUT = 60
+JOB_ID_PATTERN = re.compile(r"(?<=:\s).*_\d+")
```

Comment:
why was this changed? Could you make a comment or so about the new regex and its meaning?

Reply:
The previously used pattern (```r"\d+-\d+_\d+"```) only matched GENESIS job ids (e.g. 51000-0017_988038177) but not Regionalstatistik (e.g. 12111-31-01-4_123456789) or Zensus (e.g. 1000A-1034_123456789) IDs. The new pattern uses the fact that the genesis API returns a characteristic string when starting a job where the job id follows a colon and white space in the end (positive lookbehind assertion: ```(?<=:\s)```). The pattern ```.*_\d+``` then allows any number and combination of characters that make up the various item codes, followed by an underscore and a group of numbers, which is characteristic of a background job.